addons/mail/static/src/core/common/attachment_model.js

```diff
@@ -13,6 +13,7 @@ export class Attachment extends Record {
     extension;
     filename;
     id;
+    res_name;
```

Comment:
We try to keep the fields in alphabetical order. Please move `res_name` to where it belongs 🙏

Reply:
Between `mimetype` and `message`? :wink: